dynamic_listings/python/default/containers/fsm.py

```diff
@@ -1,20 +1,23 @@
+from typing import Callable
+
+
 class FSM:
     """
     This class defines a Finite State Machine
     The currently active state is represented by a function
     pointer
     """
 
-    current_state = None
+    current_state: Callable = None
```

Comment:
Incomplete type hint, from the top of my brain it should be something akin to `Callable[[float], None]`, but maybe it's still valid.